javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/declarations/JavaParserEnumDeclaration.java

```diff
@@ -387,7 +387,7 @@ public boolean isDefaultMethod() {
 
         @Override
         public boolean isStatic() {
-            return false;
+            return true;
```

Comment:
Can you identify the corresponding section in the JLS?

Reply:
Hi,

the section is jls-8.9.3-100-D.

see:
Java 8: https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.9.3-100-D
Java 22: https://docs.oracle.com/javase/specs/jls/se22/html/jls-8.html#jls-8.9.3-100-D